lib/src/log.rs

```diff
@@ -129,22 +130,84 @@ pub fn reset_task_log_directory(path: &Path) -> Result<(), Error> {
     Ok(())
 }
 
+pub fn read_last_lines_as_byte_deque(file: &mut File, lines: usize) -> Result<VecDeque<u8>, Error> {
```

Comment:
Missing docstring.
This function contains a quite complex algorithm. This should be well documented.

Reply:
Agreed. I'll write one up if we end up keeping this function instead of using the other crate